bootstrap/scss/_custom-styles.scss

```diff
@@ -24,10 +26,247 @@ main {
+    transform: rotate(-5deg);
+}
+
+.homepage-card-button {
+    width: fit-content;
+    padding: 5px 15px;
+}
+
+.homepage-card-info {
+    width: 60%;
+}
+
+.homepage-card-image-container {
+    width: 30%;
+    display: flex;
+    justify-content: center;
+}
+
+
+@media(max-width:1150px) {
+    .homepage-card {
+        justify-content: space-evenly;
+    }
+    .homepage-card-image-container {
+        display: none;
+    }
+}
+
+.homepage-title-container {
+    margin: 130px auto;
+}
+
+.page-header, .page-text {
+    color: #{map-get($laminas-colors, laminas)};
+}
+
+.page-header {
+    font-weight: bolder;
+    margin-bottom: 30px;
+}
+
+.homepage-support-card {
+    width: 100%;
+    position: relative;
+    margin: 2.5em auto;
+    padding: 1em;
+    background: linear-gradient(#8ABDB5, #F1F1F1);
+    border-radius: 10px;
+    font-size: 125%;
+
+    .homepage-card-info {
+        width: 50%;
+    }
+
+    .homepage-card-image-container {
+        width: 30%;
+        display: flex;
+
+        .homepage-card-image {
+            flex-shrink: 0;
+            aspect-ratio: unset;
+            width: 100%;
+        }
+    }
+
+    #crowdfunding-logo {
+        max-width: 300px;
+        position: absolute;
+        top: 1em;
+        right: 1em;
+    }
+}
+
+#support-button {
+    background: linear-gradient(to right top, #21D4FD, #0A2432);
+    i {
+        padding: .2em;
+        border-radius: 50%;
+        color: #013755;
+        background: #FFFFFF;
+    }
+}
+
+#commercial-vendor-section {
+    max-width: $max-section-width;
+}
+
+.commercial-vendor-card {
+    max-width: 350px !important;
+    border-radius: 25px !important;
+
+    .card-title {
+        color: #{map-get($laminas-colors, components)};
+        font-weight: bolder;
+    }
+
+    .homepage-card-button {
+        width: 80%;
+    }
+}
+
+// Blog section
+
+.blog-listing {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.blog-card {
+    max-width: 350px;
+    max-height: 400px;
+    border: 2px solid $primary;
+
+    .blog-card-content {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        position: relative;
+
+        .body {
+            max-height: 200px;
+            overflow: hidden;
+        }
+
+        .blog-continue-reading:before {
+            content:'';
+            width:100%;
+            height:100%;
+            position:absolute;
+            left:0;
+            bottom: 1.5em;
+            background:linear-gradient(transparent 85%, white);
+        }
+    }
+}
+
+.blog-post-content, .blog-card-content {
+    margin: 2px;
+    padding: .75em .75em .1em .75em;
+    height: 100%;
+    background: #FFFFFF;
+    border-radius: 8px;
+}
+
+.blog-post {
+    width: fit-content;
+    margin: 2em auto;
+}
+
+.blog-post-content {
+    h1, h2, h3, h4, h5, h6 {
+        color: #{map-get($laminas-colors, mezzio)};
+    }
+}
+
+#blog-pagination {
```

Comment:
The same here: use the variables and classes from Bootstrap.

https://getbootstrap.com/docs/5.3/components/pagination/